tests/forte/data/entry_type_generator_test.py

```diff
@@ -15,5 +16,37 @@ def test_get_type_attributes(self, mock_get_type_attributes):
         EntryTypeGenerator.get_type_attributes()
         self.assertLessEqual(mock_get_type_attributes.call_count, 1)
 
+    def test_get_entry_attribute_by_class(self):
+        entry_name_attributes_dict = {
+            "ft.onto.base_ontology.Sentence": [
+                "speaker",
+                "part_id",
+                "sentiment",
+                "classification",
+                "classifications",
+            ],
+            "ft.onto.base_ontology.Token": [
+                "pos",
+                "ud_xpos",
+                "lemma",
+                "chunk",
+                "ner",
+                "sense",
+                "is_root",
+                "ud_features",
+                "ud_misc",
+            ],
+            "ft.onto.base_ontology.Title": [],
+            "ft.onto.metric.SingleMetric": ["value"]
+        }
+        for entry_name in entry_name_attributes_dict.keys():
+            attribute_result = EntryTypeGenerator.get_entry_attribute_by_class(
+                entry_name
+            )
+            self.assertEqual(
+                attribute_result, entry_name_attributes_dict[entry_name]
+            )
```

Comment:
So there is nothing particularly wrong with the test cases. However, if one day we updated the ontology file, this test may fail.

One idea is that you can create some dummy example ontology class here (for example, copy the `Sentence` class here but rename it). That way we can test on those classes.

Reply:
Will refactor the test cases in the new PR.